diff --git a/node_modules/expo/build/launch/registerRootComponent.js b/node_modules/expo/build/launch/registerRootComponent.js
index 57cdc8f..80f0a9e 100644
--- a/node_modules/expo/build/launch/registerRootComponent.js
+++ b/node_modules/expo/build/launch/registerRootComponent.js
@@ -8,7 +8,7 @@ export default function registerRootComponent(component) {
         const { withDevTools } = require('./withDevTools');
         AppRegistry.registerComponent('main', () => withDevTools(component));
     }
-    if (Platform.OS === 'web') {
+    if (Platform.OS === 'web' && typeof window !== 'undefined') {
         const rootTag = document.getElementById('root') ?? document.getElementById('main');
         AppRegistry.runApplication('main', { rootTag });
     }
